scripts/zx_tests/run_tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+TESTDIR=$(dirname -- "$0")
+ZXI=${TESTDIR}/../../target/release/examples/zxi
+
+echo Running zx should-pass tests:
+for i in ${TESTDIR}/*.zx; do
+    ${ZXI} "$i" &>/dev/null
+    if [ "$?" != "0" ]; then
+        echo "[failure: should-pass] $i"
```

Comment:
Since this script appears to be going into CI, we probably want to return a non-zero code on failure.

Sorry for the drive by comment. I just noticed this while testing.